doc/api/fs.md

```diff
@@ -5362,7 +5362,7 @@ changes:
 * `buffer` {Buffer|TypedArray|DataView}
 * `offset` {integer}
 * `length` {integer}
-* `position` {integer|bigint}
+* `position` {integer|bigint|null}
```

Comment:
This parameter should be documented as optional.
```suggestion
* `position` {integer|bigint|null} **Default:** `null`
```

Reply:
Yup. Intended or not, it is optional indeed.